masterdata/configuration/programworkflows/programWorkflows_TB.csv

```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Program,Workflow concept,_order:1000
+3795a2ba-4c3e-4635-a64f-659d23cde569,,160541AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,TB Program workflow,
```

Comment:
Please check on the program UUID, this is the concept's UUID

Reply:
Updated